src/Files.App/ViewModels/MainPageViewModel.cs

```diff
@@ -15,17 +14,21 @@ namespace Files.App.ViewModels
 {
 	public class MainPageViewModel : ObservableObject
 	{
-		private IUserSettingsService userSettingsService;
-		private IAppearanceSettingsService appearanceSettingsService;
-		private readonly DrivesViewModel drivesViewModel;
-		private readonly NetworkDrivesViewModel networkDrivesViewModel;
-		private IResourcesService resourcesService;
+		private IUserSettingsService UserSettingsService { get; }
+
+		private IAppearanceSettingsService AppearanceSettingsService { get; }
+
+		private DrivesViewModel DrivesViewModel { get; }
+
+		private NetworkDrivesViewModel NetworkDrivesViewModel { get; }
+
+		private IResourcesService ResourcesService { get; }
```

Comment:
I'm personally used to not using spaces between the lines here.

Reply:
According to the guideline, section 3, item 7.

> Fields, properties, methods and constructors must be separated by one line except for property backing fields that are not separated by any amount of lines.

That is not expected. Every line that is finishing with `}` or `;` is needed to be divided with a new line break.

Reply:
Sometimes I don't like the guidelines but I guess we have to follow them.